include/SugarLogger/LoggerManager.php

```diff
@@ -119,6 +121,12 @@ public function __call(
             }
             //tell the logger to log the message
             self::$_loggers[$logger]->log($method, $message);
+
+            // Log to sentry if enabled
+            if (!empty($sugar_config['sentry']['enabled'])) {
```

Comment:
It might be useful to pass the log message's `level` along here, it seems to be supported by the client: https://docs.sentry.io/clients/php/usage/

Reply:
It does actually already know the error level, kind of. It doesn't quite match the configured log levels from the CRM, but 🤷‍♂